src/components/LangBox.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withNamespaces } from 'react-i18next';
+import { Col, Row } from 'reactstrap';
+import Select from 'react-select';
+import { options } from '../config/options';
+
+const LangBox = ({
+  handleLang,
```

Comment:
Again, `handleLang` seems to me like it should be `handleChangeLang`, as you put it against an `onChange` handler.